microsoft/testsuites/xfstests/xfstesting.py

```diff
@@ -326,14 +451,24 @@ def _execute_xfstests(
                     "Current distro doesn't support btrfs file system."
                 )
 
-        _configure_disk(
-            node,
-            data_disk,
-            {test_dev: _test_folder, scratch_dev: _scratch_folder},
-            file_system=file_system,
-        )
+        if test_type != "cifs":
+            _configure_disk(
```

Comment:
Add comments to explain what the logic is for.

Reply:
added

Reply:
The logic is not clear still. What's relation between "data_disk" and "when use azure file share"? And the "configure" should be more specific to say like "prepare disk partition" or something similar.

Reply:
updated